pipenv/cli/command.py

```diff
@@ -533,7 +533,11 @@ def graph(bare=False, json=False, json_tree=False, reverse=False):
 @argument("module", nargs=1)
 @pass_state
 def run_open(state, module, *args, **kwargs):
-    """View a given module in your editor."""
+    """View a given module in your editor, as defined by the EDITOR environment variable.
+    You can temporarily override it, for example:
+
+        EDITOR=atom pipenv open requests
+    """
```

Comment:
I think it’s better to format this like

```
View a given module in your editor.

    This uses the EDITOR environment variable. You can temporarily override it,
    for example:

        EDITOR=atom pipenv open requests
```

It is also a good idea to wrap this to under 72 characters, so the message won’t break on narrow terminals.

Reply:
OK, I've updated the formatting and squashed it into a single commit.

Yeah, I think something in the text parsing treats the paragraph as a whole no matter where we put the line break. Looks OK on my terminal I'd say:

![screenshot 2018-10-23 10 42 19](https://user-images.githubusercontent.com/1818628/47347952-aa9ffd00-d6b0-11e8-903a-c642f08c94bb.png)

Reply:
Hmm alright then